tracer/src/Datadog.Trace/ClrProfiler/AutoInstrumentation/AspNetCore/UserEvents/AuthenticationHttpContextExtensionsIntegration.cs

```diff
@@ -65,16 +65,29 @@ internal static object OnAsyncMethodEnd<TTarget>(object returnValue, Exception e
                 setTag(Tags.AppSec.EventsUsers.LoginEvent.SuccessAutoMode, Security.Instance.Settings.UserEventsAutomatedTracking);
                 foreach (var claim in claimsPrincipal.Claims)
                 {
-                    if (claimsToTest.Contains(claim.Type))
+                    if (security.Settings.UserEventsAutomatedTracking == SecuritySettings.UserTrackingExtendedMode)
```

Comment:
ultra nit, but to make this easier to read you could add an helper function in SecuritySettings. Something like:
`bool IsExtendedTrackingActivated()`

Reply:
added in 139bd3e25b6c4577effbbe14880302003ad5de8e

Reply:
Nice thanks a lot